internal-client/src/main/java/com/tencent/rss/client/response/RssAccessClusterResponse.java

```diff
@@ -19,8 +19,25 @@
 package com.tencent.rss.client.response;
 
 public class RssAccessClusterResponse extends ClientResponse {
+  private final String storageType;
+  private final String storagePath;
 
-  public RssAccessClusterResponse(ResponseStatusCode statusCode, String messge) {
+  public RssAccessClusterResponse(ResponseStatusCode statusCode, String message) {
+    this(statusCode, message, "", "");
```

Comment:
default storageType

Reply:
These empty strings are not for the default value, it's because we don't need to put any conf info in the response if the access is denied.